book/urban/sensors/urban-sensors-ukv.ipynb

```diff
@@ -0,0 +1,397 @@
+{
```

Comment:
<p>I suggest to add the Note after <strong>Authors </strong>using<strong> </strong>the following Markdown:</p><pre class="ql-syntax" spellcheck="false">:::{note}
Note this data should be used only for non-commercial purposes!.  
:::
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/acocac/environmental-ai-book/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>This markdown does not work for me in my Jupyter notebook ! is it some kind of special nb extension?</p>

Reply:
<p>Sam, most of Jupyter Book’s functionality comes from Sphinx extensions. This means if you want to see the above markdown for a given ipynb, you'll need to install the Sphinx plugin in your python interpreter. </p><p> </p>